testing/cli/test_cli_sct_register_multimodal.py

```diff
@@ -92,7 +92,7 @@ def test_sct_register_multimodal_with_softmask(tmp_path):
 
     NB: For 'gaussian', ANTs binaries can't handle softmasks natively, so SCT should be applying
         the mask directly to the image. Related links:
-            * https://github.com/ivadomed/pipeline-hemis/issues/3.
+            * github.com/ivadomed/pipeline-hemis/issues/3.
```

Comment:
Same question as above, why remove the https?

Reply:
Originally, I modified this to also exclude it from the link-checking, since it's a private GH repo. But, then I switched over to excluding the actual `pipeline-hemis` string... and then forgot to reinstate the `https://`. :facepalm: